### DISCUSSION TITLE
Action modal to prevent close when click away

### DISCUSSION THREAD

**Discussion by Pamposgsk** [laravel/nova-issues#4312] [Q&A] [answered]
Hi,
Is there a possibility to customise an action modal to prevent click away close.
I would like to be able to close modal only if I click the cancel button or if I run the action.
Sometimes the action modal has to much information so closing the modal accidentally is really time consuming
Thanks 
Pampos Hadjimichael

**Comment by davidhemphill** [comment, marked as answer]
The action modal doesn't support click away in v4. The only way to dismiss the modal is to press escape or click the cancel button.

**Comment by Pamposgsk** [comment]
Thanks a lot.Great news!
It seems I need to upgrade to v4 😀😀
